### PR TITLE
feat(TMRX-1314): add credits lead image articles

### PR DESCRIPTION
**Pull request by ibrahim-kurhan**
### Description

add credits lead image articles

[JIRA-1314](https://nidigitalsolutions.jira.com/browse/TMRX-1314)

### Checklist

- [x] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):
Screenshot 2023-07-26 at 19 05 47

### CODE REVIEW

File: packages/ts-newskit/src/components/slices/lead-article/index.tsx [hunk 1]
```diff
@@ -139,6 +141,7 @@ export const LeadArticle = ({
                 typographyPreset="utilityMeta010"
               >
                 {images && images.caption}
+                {images && <StyledSpan>{images.credits}</StyledSpan>}
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/slices/lead-article/index.tsx:144]: Same here - not sure if credits always come through in the data. Sometimes can be null from what I remember.

Comment (bibicollins1) [reply packages/ts-newskit/src/components/slices/lead-article/index.tsx:144]: Oh actually unless your clearCreditsAndCaption function returns them as empty strings that should be fine I think

**Review by bibicollins1 — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/lead-article/index.tsx (hunk 1 above)
Comment (douglasmik) [reply packages/ts-newskit/src/components/slices/lead-article/index.tsx:144]: Ideally, would have check here also to not return empty element if string is empty

**Review by douglasmik — APPROVED** [review]

File: packages/ts-newskit/src/components/slices/lead-article/index.tsx (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/slices/lead-article/index.tsx]: added extra check also i have changed hasCaption to hasCaptionOrCredits and checking both

**Pull request merged**